src/protectionmarketlist.schema.json

```diff
@@ -0,0 +1,447 @@
+            }
+          }
+        }
+      }
+    },
+    "ProtectionMarketInfo": {
+      "type": "object",
+      "description": "Metadata for a single market in a protection market list",
+      "additionalProperties": false,
+      "properties": {
+        "chainId": {
+          "type": "integer",
+          "description": "The chain ID of the Ethereum network where this token is deployed",
+          "minimum": 1,
+          "examples": [1, 42]
+        },
+        "address": {
+          "type": "string",
+          "description": "The checksummed address of the protection market on the specified chain ID",
+          "pattern": "^0x[a-fA-F0-9]{40}$",
+          "examples": ["0xdBDF2fC3Af896e18f2A9DC58883d12484202b57E"]
+        },
+        "decimals": {
+          "type": "integer",
+          "description": "The number of decimals the market's receipt token has",
+          "minimum": 0,
+          "maximum": 255,
+          "examples": [18]
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the protection market",
+          "minLength": 1,
+          "maxLength": 100,
+          "pattern": "^[ \\w.'+\\-%\\/À-ÖØ-öø-ÿ:]+$",
+          "examples": ["Cozy USD Coin"]
+        },
+        "symbol": {
+          "type": "string",
+          "description": "The symbol for the protection market; must be alphanumeric",
+          "pattern": "^[a-zA-Z0-9+\\-%\\/\\$]+$",
+          "minLength": 1,
+          "maxLength": 100,
+          "examples": ["cozyUSDC"]
+        },
+        "logoURI": {
+          "type": "string",
+          "description": "A URI to the protection market's logo asset; if not set, interface will attempt to find a logo based on the token address; suggest SVG or PNG of size 256x256",
+          "format": "uri",
+          "examples": ["ipfs://QmXfzKRvjZz3u5JRgC4v5mGVbm9ahrUiB4DgzHBsnWbTMM"]
+        },
+        "investmentOpportunity": {
+          "$ref": "#/definitions/InvestmentOpportunity"
+        },
+        "isTriggered": {
+          "type": "boolean",
+          "description": "True if this market has been triggered, false if not triggered or not applicable",
+          "examples": [true, false]
+        },
+        "trigger": {
+          "type": "string",
+          "description": "The checksummed address of the market's trigger contract, or the zero address for markets without a trigger",
+          "pattern": "^0x[a-fA-F0-9]{40}$",
+          "examples": [
+            "0x02E57c5800a345Fe97d73fA8bB72B948E19aA499",
+            "0x0000000000000000000000000000000000000000"
+          ]
+        },
+        "underlying": {
+          "$ref": "#/definitions/TokenInfo"
+        },
+        "tags": {
+          "type": "array",
+          "description": "An array of tag identifiers associated with the market; tags are defined at the list level",
+          "items": {
+            "$ref": "#/definitions/TagIdentifier"
+          },
+          "maxItems": 10,
```

Comment:
Just wondering why the max and why 10?

Reply:
That's how Uniswap has it and I just left it since I didn't see a strong reason to change it 🤷